internal/driver/driver.go

```diff
@@ -176,6 +178,12 @@ func applyCommandOverrides(cmd []string, v variables) variables {
 			v.set("nodecount", "80")
 		}
 	}
+
+	if outputFormat == report.Proto {
+		trim, tagfilter, filter = false, false, false
```

Comment:
Perhaps for future unless there is an easy way to do it: it would be good to have a way to generate reports in the proto format keeping the filter. E.g. using the internal -flame command with a -tagfocus filter to upload a subset of the profile.

Reply:
I'm assuming that this could be done just by not setting `filter` and `tagfilter` to false. Not sure if we would want to also, for example, add a comment to the profile to indicate filtering was done.

Will experiment, then look into changing.

Reply:
This can be done later.